src/helpers/bufferToHex.ts

```diff
@@ -0,0 +1,3 @@
+export const bufferToHex = function (buf: Buffer): string {
```

Comment:
This is too small for it's own file, would rather it be in `tools.ts` 

Similar this could be reorganized